modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To query the cluster resource metrics API.
+* To monitor or manage alerts and silences.
+* To access Alertmanager component.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
+
+|===
+|Name |Description
+
+|`cluster-monitoring-metrics-api` |Users with this role have the ability to query the cluster resource metrics API for core platform monitoring.
+
+|`cluster-monitoring-operator-alert-customization` |Users with this role can monitor `AlertingRule` and `AlertRelabelConfig` resources for core platform monitoring. These permissions are required for the alert customization feature.
+
+|`monitoring-alertmanager-edit` |Users with this role can create and silence alerts for core platform monitoring in the *Administrator* perspective of the {product-title} web console.
```

Comment:
It also provides access to the Alertmanager web server on port 9094 https://docs.openshift.com/container-platform/4.17/observability/monitoring/accessing-third-party-monitoring-apis.html#openshift-monitoringalertmanager-main
```suggestion
|`monitoring-alertmanager-edit` |Users with this role can manage alert silences for core platform monitoring in the *Administrator* perspective of the {product-title} web console.
```